cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go

```diff
@@ -55,16 +55,16 @@ func runCleanupNode(c workflow.RunData) error {
 	certsDir := r.CertificatesDir()
 
 	// Try to stop the kubelet service
-	klog.V(1).Infoln("[reset] Getting init system")
+	klog.V(1).Infoln("[reset] getting init system")
 	initSystem, err := initsystem.GetInitSystem()
 	if err != nil {
-		klog.Warningln("[reset] The kubelet service could not be stopped by kubeadm. Unable to detect a supported init system!")
-		klog.Warningln("[reset] Please ensure kubelet is stopped manually")
+		klog.Warningln("[reset] WARNING: The kubelet service could not be stopped by kubeadm. Unable to detect a supported init system!")
```

Comment:
Why add the extra warning, there will already be a [WARNING] prefix on the message.